packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-load/aspnet/integrationsteps.md

```diff
@@ -1,4 +1,4 @@
-### Step 1: Call StartWidgetSigninAsync
+### Step 1: Navigate to sign-in page
 
 #### Summary
```

Comment:
This step has "Summary" and "Code sample" subheadings, which I've not seen in any other use cases. I'd delete them for consistency.

Reply:
subheadings removed.